modules/dedicated-cluster-install-deploy.adoc

```diff
@@ -19,19 +19,27 @@ Cluster Logging Operator and Elasticsearch Operator.
 
```

Comment:
I might change the previous note to explain that deploying the logging operator also deploys the elasticsearch operator.